--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -1744,7 +1744,19 @@
  perlasm_scheme => "mingw64",
  shared_rcflag => "--target=pe-x86-64",
  multilib => "64",
  },
+
+ "mingwarm64" => {
+ inherit_from => [ "mingw-common" ],
+ cflags => "",
+ sys_id => "MINGWARM64",
+ bn_ops => add("SIXTY_FOUR_BIT"),
+ asm_arch => 'aarch64',
+ uplink_arch => 'armv8',
+ perlasm_scheme => "win64",
+ shared_rcflag => "",
+ multilib => "-arm64",
+ },
 
 #### UEFI
  "UEFI" => {
